content/blog/simple-reproducible-kubernetes-deployments/index.md

```diff
@@ -24,7 +24,7 @@ powerful, Kubernetes deployment workflows.
 
 ## Less YAML, More Robustness
 
-[Pulumi 0.15]({{< relref "/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more.md" >}})
+[Pulumi 0.15]({{< relref "/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more/index.md" >}})
```

Comment:
Nit, but I think you could just do this:

```suggestion
[Pulumi 0.15]({{< relref "/blog/announcing-pulumi-0.15-kubernetes-cicd-openstack-and-more" >}})
```

... which would be a smidge more consistent. But not a big thing at all.

Reply:
I just tried your suggestion and it caused Hugo (0.81) to throw a `REF_NOT_FOUND` error when running the site locally. I assume this is because the index pages for blogs don't have the `_` prefix in the file name?